raveandroid/src/main/java/com/flutterwave/raveandroid/card/CardFragment.java

```diff
@@ -849,4 +832,58 @@ public void onPageFinished(WebView view, String url) {
         }
     }
 
+    private class ExpiryWatcher implements TextWatcher {
+
+        private String lastInput = "";
+
+        @Override
+        public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+        }
+
+        @Override
+        public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+        }
+
+        @Override
+        public void afterTextChanged(Editable editable) {
+            String input = editable.toString();
+            SimpleDateFormat simpleDateFormat = new SimpleDateFormat("MM/yy");
+            Calendar calendar = Calendar.getInstance();
```

Comment:
Nice one man, learnt something new :smile: 

You should consider  creating a static `Calendar` as  cost of calling `Calendar.getInstance` every time text is changed is quite high [see](http://cephas.net/blog/2006/02/25/the-cost-of-calendar-object-creation/) [see](http://www.informit.com/articles/article.aspx?p=1216151&seqNum=5)

Reply:
Oh...true.

I will fix that. Thanks

Reply:
It's updated now.